code/utils/preprocess.py

```diff
@@ -396,8 +396,8 @@ def motion_correct(dff, fs=20, M=TukeyBiweight(1)):
             Preprocessed fiber photometry signal with motion correction applied
             (dF/F + motion correction).
     """
-    sos = butter(N=2, Wn=0.3, fs=fs, output="sos")
-    dff_filt = sosfilt(sos, dff, axis=0).T
+    b, a = butter(N=2, Wn=0.3, fs=fs)
+    dff_filt = filtfilt(b, a, dff, axis=0).T
```

Comment:
If you think sos is better, using `sosfiltfilt` maybe a better option.
But up to you, rather a question from curiosity and no intention to nitpick! 
If this is ok for you as it is, let's merge to main.

Reply:
Yes, the docs say 'sos' should be used for general-purpose filtering instead of the default 'ba', thus changed all the `filtfilt` to `sosfiltfilt`. sos breaks high-order filters into second-order sections, but our second-order filter is already in its simplest form, thus the two representations should actually be identical

Reply:
ok, great. thanks for clarification. I will merge the current ver to main, then.